test/helpers/save-screenshot.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
```

Comment:
Please give the file a name matching the pattern of the WikibaseApiPatch filename for consistency.